### DISCUSSION TITLE
Execute click without pressing through touchpad

### DISCUSSION THREAD

**Discussion by Septias** [leftwm/leftwm#955] [Q&A]
Hi, 

on my laptob I always have to click through on the touchpad instead of just touching it. In Gnome that works. I'm not sure how to configure that. Is that a leftwm property or do I have to configure it elsewhere? Also I would like to change the scroll-behaviour.

**Comment by hertg** [comment]
It might be that Gnome (and other desktop environments for that matter) comes with touchpad configs for X11, which LeftWM (and other minimal window managers) don't. You will probably find some useful information about touchpad configuration in the Arch wiki: https://wiki.archlinux.org/title/Touchpad_Synaptics

For additional reference, here's the one I'm using. It includes configs to make tap click work, two-finger tap for right-click, three-finger tap for middle-click, and inversed two finger scrolling (By default the two finger scrolling is probably not what you want, because moving the fingers up scrolls upwards instead of down).
https://github.com/hertg/deploy/blob/master/roles/base/files/systemconfig/X11/70-synaptics.conf

**Comment by 0323pin** [comment]
Just adding to what @hertg already said but, if you are using `libinput` instead of `synaptics`, I have two laptops using LeftWM, one on `synaptics` and one on `libinput`.

Create the file `/etc/X11/xorg.conf.d/40-libinput.conf` with the following content:

```
Section "InputClass"
	Identifier "libinput touchpad catchall"
	MatchIsTouchpad "on"
	MatchDevicePath "/dev/input/event*"
	Driver "libinput"
	Option "Tapping" "on"
	Option "NaturalScrolling" "true"
	Option "ClickMethod" "clickfinger"
	Option "TappingButtonMap" "lrm"
	Option "DisableWhileTyping" "true"
EndSection
```
